_rules/text-contrast-enhanced-09o5cg.md

```diff
@@ -65,6 +67,8 @@ Passing this rule does not mean that the text has sufficient color contrast. If
 
 When the text color or background color is not specified in the web page, colors from other [origins][] will be used. Testers must ensure colors are not affected by styles from a [user origin][], such as a custom style sheet. Contrast issues caused by specifying the text color but not the background or vice versa, must be tested separately from this rule.
 
+This rule is closely related to [success criterion 1.4.3 Contrast (Minimum)][sc143]. Because this rule is stricter, text that passes this rule will likely satisfy 1.4.3 Contrast (Minimum).
```

Comment:
Is "likely" necessary here? When would passing 1.4.6 not pass 1.4.3? The phrasing sort of begs the question.